soco/core.py

```diff
@@ -918,11 +918,15 @@ def unjoin(self):
         zone_group_state_shared_cache.clear()
         self._parse_zone_group_state()
 
-    def switch_to_line_in(self):
+    def switch_to_line_in(self, source=None):
         """ Switch the speaker's input to line-in.
 
+        Args:
+            source (SoCo): The speaker whose line-in should be played.
+                Default is line-in from the speaker itself.
+
         Returns:
```

Comment:
This part of the docstring appears to obsolete. We switched a ling time ago away from returning booleans in indicate success. I know you didn't introduce it, but might as well update while we are at it. You can just delete the returns part.